mms/model_server.py

```diff
@@ -70,6 +70,7 @@ def start():
 
         mms_home = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
         cmd = [java, "-Dmodel_server_home={}".format(mms_home)]
+        #cmd.append("-Dio.netty.leakDetection.level=ADVANCED")
```

Comment:
I don't think this is required. If you think we should have this, enable this when "debug=1" is enabled.